### PR TITLE
Judicial System: Displays a special message if user cancels electronic signature

### PR DESCRIPTION
**Pull request by gudjong**
# Electronic Signature Cancel Message

## What

Displays a special cancel message if the user cancels an electronic signature.

## Why

This is part of the requirements.

image

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against master before asking for a review

### CODE REVIEW

File: apps/judicial-system/api/src/app/modules/case/case.controller.ts [hunk 1]
```diff
@@ -255,7 +273,22 @@ export class CaseController {
       )
     }
 
-    return this.caseService.confirrmSignature(existingCase, documentToken)
+    try {
+      const response = await this.caseService.confirrmSignature(
```
Comment (oddsson) [inline apps/judicial-system/api/src/app/modules/case/case.controller.ts:277]: Typo... confirrmSignature 🖌 confirmSignature

File: apps/judicial-system/web/src/routes/Judge/Confirmation/Confirmation.tsx [hunk 2]
```diff
@@ -406,15 +406,17 @@ export const Confirmation: React.FC = () => {
       {modalVisible && (
         <Modal
           title={
-            confirmSignatureResponse?.httpStatusCode > 400
-              ? 'Auðkenning tókst ekki'
+            confirmSignatureResponse?.httpStatusCode >= 400
+              ? confirmSignatureResponse?.code === 7023
```
Comment (oddsson) [inline apps/judicial-system/web/src/routes/Judge/Confirmation/Confirmation.tsx:410]: Consider creating an enum for dokobit error codes

**Review by oddsson — CHANGES_REQUESTED** [review]

File: apps/judicial-system/api/src/app/modules/case/case.controller.ts (hunk 1 above)
Comment (gudjong) [reply apps/judicial-system/api/src/app/modules/case/case.controller.ts]: Fixed

File: apps/judicial-system/web/src/routes/Judge/Confirmation/Confirmation.tsx (hunk 2 above)
Comment (gudjong) [reply apps/judicial-system/web/src/routes/Judge/Confirmation/Confirmation.tsx]: Just commented for now.

**Review by oddsson — APPROVED** [review]

**Pull request merged**